source/computer/macos-setup.rst

```diff
@@ -122,10 +123,17 @@ Homebrew
    在 Apple M1 芯片的 Mac 下安装 Homebrew 后，还需要执行如下命令以配置 Homebrew::
 
        $ echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> ~/.zshrc
-       $ source ~/.zshrc
+       $ eval "$(/opt/homebrew/bin/brew shellenv)"
 
-使用
-""""
+添加其他仓库
+""""""""""""
+
+::
+
+    brew tap homebrew/cask
```

Comment:
We also need to set the USTC mirror for `homebrew cask` and `homebrew bottles` (https://brew.idayer.com/guide/change-source/).

```
brew tap homebrew/cask
git -C "$(brew --repo homebrew/cask)" remote set-url origin https://mirrors.ustc.edu.cn/homebrew-cask.git
echo 'export HOMEBREW_BOTTLE_DOMAIN=https://mirrors.ustc.edu.cn/homebrew-bottles/bottles' >> ~/.zshrc
source ~/.zshrc
brew update
```

Reply:
```
brew tap homebrew/cask
git -C "$(brew --repo homebrew/cask)" remote set-url origin https://mirrors.ustc.edu.cn/homebrew-cask.git
```
These two commands won't work, because `brew tap homebrew/cask` clones the homebrew/cask repository from GitHub, which will fail if github is inaccessible. We should use the following command instead:
```
brew tap --custom-remote --force-auto-update homebrew/cask https://mirrors.ustc.edu.cn/homebrew-cask.git
```
which clones the homebrew/cask repository from the custom git URL `https://mirrors.ustc.edu.cn/homebrew-cask.git` and also sets the git remote URL. See http://mirrors.ustc.edu.cn/help/homebrew-cask.git.html.